### DISCUSSION TITLE
how can i configure paru to clone git repos using ssh instead of https

### DISCUSSION THREAD

**Discussion by ingenarel** [Morganamilo/paru#1222] [Q&A]
i have a really unstable internet connection. and cloning over https fails 99% of the time. however i changed a few git configs and now if i clone git repos using ssh, they all clone perfectly, altho it does take a long time.

so is there any way i can do that?

**Comment by Morganamilo** [comment]
If you're still stuck on this, Paru doesn't have a feature for this but you can just add 

```
[url "ssh://aur.archlinux.org/"]
	insteadOf = https://aur.archlinux.org/
```

to your `~/.gitconfig`.
